pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -386,3 +389,116 @@ def get_convergence(self):
     def get_fermi(self):
         pattern = r"Fermi energy:\s+(\d+\.\d+)\s+eV"
         return np.array(re.findall(pattern, self.log_main)).astype(float)
+
+
+class SphinxWavesParser:
+    """ Class to read SPHInX waves.sxb files (HDF5 format)
+    
+        Initialize with waves.sxb filename, or use load ()
+    
+    """
+    
+    def __init__(self, file_name="waves.sxb", cwd="."):
+        """
+        Args:
+            file_name (str): file name
+            cwd (str): directory path
+        """
+        if Path(file_name).is_absolute():
+            self.load(Path(file_name))
+        else:
+            path = Path(cwd) / Path(file_name)
+            self.load(path)
+        
+        
+
+    def load(self, filename):
+        """ Load a waves.sxb file (HDF5 format)
+        
+            filename: file name
+        """
+        self.wfile = h5py.File (filename)
+        self._eps = None
```

Comment:
In principle all variables should be defined in `__init__` (e.g. cf. [this page](https://stackoverflow.com/questions/20661448/should-all-member-variables-be-initialized-in-init)). so `self._eps = None` should also be there. Then maybe we don't need `load` since it has only one line if it's anyway used only once.